2022/23xxx/CVE-2022-2385.json

```
@@ -0,0 +1,109 @@
+{
+    "CVE_data_meta": {
+        "ASSIGNER": "security@kubernetes.io",
+        "DATE_PUBLIC": "2022-07-11T16:00:00.000Z",
+        "ID": "CVE-2022-2385",
+        "STATE": "PUBLIC",
+        "TITLE": "AccessKeyID validation bypass"
+    },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "aws-iam-authenticator",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": ">",
+                                            "version_value": "v0.5.1"
+                                        },
+                                        {
+                                            "version_affected": "<=",
+                                            "version_value": "v0.5.9"
+                                        }
```

So this affects v0.5.2 - v0.5.9?

nit: slight preference to use `>=` on the bottom end, and `<` on the upper, so it would be:
```suggestion
                                        {
                                            "version_affected": ">=",
                                            "version_value": "v0.5.2"
                                        },
                                        {
                                            "version_affected": "<",
                                            "version_value": "v0.5.10"
                                        }
```

But it doesn't really matter...

---

Actually, looking at the linked issue it looks like the upper bound is off-by-one?

---

Good catch, the fix version is v0.5.9